tests/dummy/config/ember-try.js

```diff
@@ -5,20 +5,66 @@ const { embroiderSafe, embroiderOptimized } = require('@embroider/test-setup');
 
 module.exports = async function () {
   return {
+    useNpm: true,
     scenarios: [
       {
         name: 'ember-lts-3.20',
         npm: {
           devDependencies: {
-            'ember-source': '~3.20.5',
+            'ember-resolver': '^8.0.0',
+            'ember-source': '~3.20.0',
+            '@ember/test-helpers': '^2.9.3',
+            'ember-page-title': '^7.0.0',
+            'ember-qunit': '^5.1.4',
           },
         },
       },
       {
         name: 'ember-lts-3.24',
         npm: {
           devDependencies: {
-            'ember-source': '~3.24.3',
+            'ember-resolver': '^8.0.0',
+            'ember-source': '~3.24.0',
+            '@ember/test-helpers': '^2.9.3',
+            'ember-page-title': '^7.0.0',
+            'ember-qunit': '^5.1.4',
+          },
+        },
+      },
+      {
+        name: 'ember-lts-3.28',
```

Comment:
We can make this a major release and drop support for these unsupported ember versions, testing should start at `4.4`. If it happens to work for older ember that's ok too, but we don't need to guarantee, or test for, support.

Reply:
Okay, we ca do it... my plan was to make nothing braking, bacause, so we can fix FA5 at least in v1.0, beause all latest versions of this addon do not anymore support FA5

Reply:
This is a major version bump as is because of the node drop. So it can't really go into v1 anyway.

Reply:
Okay, i will try to make this changes tomorow